account-decoder/src/lib.rs

```diff
@@ -66,6 +73,19 @@ impl UiAccount {
                 base64::encode(slice_data(&account.data, data_slice_config)),
                 encoding,
             ),
+            UiAccountEncoding::Base64Zstd => {
+                let mut encoder = zstd::stream::write::Encoder::new(Vec::new(), 0).unwrap();
```

Comment:
For system accounts, base64+zstd is actually larger than than base64.   But I didn't bother getting fancy with more fallbacks to base64, figured it wasn't worth it